distribution/src/bin/elasticsearch-env

```diff
@@ -119,16 +119,23 @@ if [[ "$ES_DISTRIBUTION_TYPE" == "docker" ]]; then
 
   declare -a es_arg_array
 
+  containsElement () {
```

Comment:
elasticsearch-env gets invoked twice in the docker build, so it duplicates the env variables as cmd line parameters.